src/timeit_gtkmm/GUI/DetailsDialog.cpp

```diff
@@ -103,6 +107,7 @@ DetailsDialog::DetailsDialog(
 			&DetailsDialog::on_change));
 	stopTimeMinute.signal_value_changed().connect(sigc::mem_fun(this,
 			&DetailsDialog::on_change));
+	m_taskAccessor.attach(this);
```

Comment:
This observer attach in the constructor is not matched in the destructor. This causes a segfault after  closing the details dialog.

Reply:
Fixed in 56518a40770060827dfd173dd0bd79bfcfe7b6ef